src/System.Management.Automation/FormatAndOutput/common/FormatViewManager.cs

```diff
@@ -550,7 +504,7 @@ internal static bool IsPropertyLessObject(PSObject so)
             else
             {
                 if (DefaultScalarTypes.IsTypeInList(typeNames) ||
-                    IsPropertyLessObject(so))
+                    !HasNonRemotingProperties(so))
```

Comment:
Common pattern here & elsewhere seems to be to put the boolean operators at the beginning of a continued line.